pkg/apiserver/compression.go

```diff
@@ -3,18 +3,21 @@ package apiserver
 import (
 	compressGzip "compress/gzip"
 	"fmt"
-	ginGzip "github.com/gin-contrib/gzip"
-	"github.com/gin-gonic/gin"
 	"io"
 	"strconv"
+
+	ginGzip "github.com/gin-contrib/gzip"
+	"github.com/gin-gonic/gin"
 )
 
+// CompressionsSettings for the API.
 type CompressionSettings struct {
 	Level         string                     `cfg:"level" default:"default" validate:"oneof=none default best fast 0 1 2 3 4 5 6 7 8 9"`
 	Decompression bool                       `cfg:"decompression" default:"true"`
 	Exclude       CompressionExcludeSettings `cfg:"exclude"`
 }
 
+// CompressionExcludeSettings for this API.
```

Comment:
Code comments be like 
![cat comment](https://user-images.githubusercontent.com/16260637/145575860-518a2e76-a185-4c79-a027-59bd332354c5.png)